collections/s5p-no2-tropno-daily-check.yaml

```diff
@@ -4,6 +4,7 @@ AdditionalInfoExternal:
     Title: Additional info
     Path: s5p-no2-tropno-daily-check/README.MD
 Image: s5p-no2-tropno-daily-check/s5p-no2-tropno-daily-check.png
+EDCBrowser: https://browser.eurodatacube.com/?zoom=7&lat=53.23106&lng=-9.38391&collectionId=s5p-no2-tropno-daily-check&customVisualizationSelected=true&evalscript=Ly9WRVJTSU9OPTMKZnVuY3Rpb24gc2V0dXAoKSB7CiAgcmV0dXJuIHsKICAgIGlucHV0OiBbInRyb3BubzIiXSwKICAgIG91dHB1dDogeyBiYW5kczogMywgc2FtcGxlVHlwZTogIkFVVE8iIH0KICB9Owp9CgpmdW5jdGlvbiBldmFsdWF0ZVBpeGVsKHNhbXBsZSkgewogIHJldHVybiBbMi41KnNhbXBsZS50cm9wbm8yLDIuNSpzYW1wbGUudHJvcG5vMiwyLjUqc2FtcGxlLnRyb3BubzJdO30%3D&type=sentinel-hub-edc&fromTime=2021-03-08T09%3A44%3A48.897Z&toTime=2022-03-08T09%3A44%3A48.897Z#custom-script
```

Comment:
There is a typo in this [folder name](https://github.com/eurodatacube/public-collections/blob/main/collections/s5p-no2-tropno-daily-check/scipt.js) (scipt --> script) leading to the script/visualisation not being correctly loaded. I suggest to fix this also here and then we can use the default visualization in the link. 

`https://browser.eurodatacube.com/?zoom=5&lat=50.44351&lng=-5.75684&collectionId=s5p-no2-tropno-daily-check&layerId=NO2&type=sentinel-hub-edc&fromTime=2021-03-08T09%3A44%3A48.897Z&toTime=2022-03-08T09%3A44%3A48.897Z`

Reply:
Fix the filename and use the default visualisation